docs/walkthrough.rst

```diff
@@ -0,0 +1,199 @@
+  namespace (any random input will work). This does not affect tags.
+
+
+Importing annotations
+---------------------
+
+Starting from the file we exported as explained in the previous section, \
+we proceed to edit it within a spreadsheet editor, adding more columns to the csv, \
+including a row specifying the namespace to assign for each key:
+
+.. figure:: images/annotation_05_KV_to_import.png
+
+   *Result of populating the csv shown in a text editor*
+..
+
+We added several columns to annotate our dataset with Key-Value pairs \
+following the `REMBI <https://doi.org/10.1038/s41592-021-01166-8>`_ guidelines \
+(after saving the document, our ``TAB`` separator became ``,``).
+
+We proceed and start the script "*Import from CSV*".
+
+.. figure:: images/annotation_06_script_import.png
+
+   *Selection of all the datasets of a project and the Import from CSV script.*
+..
+
+There are many parameters here that could help you fine tune the way to import annotations. More \
+  on that in the :doc:`parameter description section</parameter_description>` of this guide.
+
+
+We can then see in the OMERO activities that the Key-Value pairs were added to 5 \
+datasets out of the 11 present in this project (as expected). We also see three set of \
+key-value pairs, one for each namespace we used in the csv.
+
+.. figure:: images/annotation_07_KV_import_printout.png
+
+  *The script output (5 entries in the csv matched to 5 dataset out of 11) and the\
+  resulting key-value pairs annotation.*
+..
+
+Converting the Key-Value pairs namespace
+----------------------------------------
+
+Key-Value pairs are assigned a category/label (known as namespace). \
+This grants flexibility so that different annotations can be \
+distinguished or isolated (like for exporting/deleting only those with a given \
+namespace).
+
+   In fact, if you created Key-Value pairs in OMERO.web, you have used \
+   namespaces without noticing it: OMERO assigns by default the \
+   "client namespace" (``openmicroscopy.org/omero/client/mapAnnotation`` in full)\
+   , a special namespace recognized by OMERO.web allowing to edit them.
+
+We would like to do a few changes in OMERO.web, and for that we would need to \
+change the namespace to the default OMERO.web namespace. We indicate the namespace we \
+want to edit and leave the second field empty (default namespace is then used).
+
+.. figure:: images/annotation_08_convert_namespace.png
+
+  *The script to convert the namespace of key-value pairs annotations.*
+..
+
+And here is our five Key-Value pairs annotations with converted namespace:
+
+.. figure:: images/annotation_09_converted_KV.png
+
+  *The script output (5 dataset had annotations with the default namespace) and the\
+  resulting key-value pairs annotation.*
+..
+
+Deleting Key-Value pairs
+------------------------
+
+It turns out that there are more than a few edits to do on our key-value pairs, and \
```

Comment:
It seems that the first use of the deletion was obvious. 2 consecutive imports or, more notoriously, an export followed by edit and reimport, will end with a duplicated KVPs on objects (at least that was our experience during workshop). Hence, we used the deletion to ``clean up`` the KVPs just after export and then reimport.
It is dawning on me slowly that you are assuming some namespace workflow which should replace the workflow I just indicated ? I would not underestimate the adverse reaction of the user to any ``namespace`` parameter, which is almost certainly unheard/unclear to start with.